### ISSUE TITLE
Make medics killable again?

### ISSUE THREAD

**Issue by Tormentor667** [Realm667/WolfenDoom#1477]
![grafik](https://github.com/user-attachments/assets/4b9256da-2f70-4080-9492-c4525938cc9b)

This is a discussion. I am playing through the mod again and kinda find it strange that medics are not killable. I know we had a huge flamewar on Doomworld regarding this, but ingame, it feels more unrealistic. I think the player should be given the decision what to do with them.

They can surrender, yes, but the player should still be able to deal with them how he wants it.

What do you think?

**Comment by hawkwind3** [comment]
For "realism" I guess they should be killable once surrendered. But I'm happy either way.

**Comment by AFADoomer** [comment]
The discussion on this is documented here: https://github.com/Realm667/WolfenDoom/issues/322

They are unarmed medics wearing the Red Cross.  Don't let the player do things that would be obvious war crimes in the real world.

**Comment by hawkwind3** [comment]
Fair enough. For consistency, and to not have BJ charged, all surrendered characters should not be killable then.

**Comment by Tormentor667** [comment]
> The discussion on this is documented here: #322
> 
> They are unarmed medics wearing the Red Cross. Don't let the player do things that would be obvious war crimes in the real world.

Yeah… good point, I forgot the n but we made the decision for a reason. But I find it kinda distracting that they stay forever. Couldn’t we make them disappear after a while? E.g. fading them out after 10 seconds of surrendering?

**Comment by hawkwind3** [comment]
Really, all surrendered players then should be non-killable. As @AFADoomer stated " Don't let the player do things that would be obvious war crimes in the real world."

**Comment by Tormentor667** [comment]
A fix has been pushed to master in commit d0cbda0eb (https://github.com/Realm667/WolfenDoom/commit/d0cbda0eb): surrendering medics are no longer invulnerable and may still disappear after leaving the player's line of sight. Please test this in the current master build and confirm that the resulting behavior matches the intended functionality for this issue.

**Comment by hawkwind3** [comment]
@Tormentor667 - maybe this can be closed ?

**Comment by AFADoomer** [comment]
Re-stating and documenting my formal objection to allowing the killing of medics here for posterity.......

**Comment by hawkwind3** [comment]
My research reveals that German medics in WW2 were given weapons. So, is this possible...

Give the medics a pistol, and if they use it, like the mechanics and scientists, then make it legal to kill them, but until then, to not be able to kill them. A good compromise I think. And, of course, not kill a surrendered mechanic/scientist. Is this not currently a war crime ?

So what I'm saying is this ... all surrendered enemy should not be killable. Cannot have it both ways IMO.

**Comment by Tormentor667** [comment]
> Re-stating and documenting my formal objection to allowing the killing of medics here for posterity.......

As I have full respect of your opinion, I support leaving the mechanic as is and reversing my past decision to make them killable again.

**Comment by Tormentor667** [comment]
Fixed and pushed to master in af02a8091, following the latest design decision in this discussion to reverse the killable-medic change. Medics are protected before and after surrender. A player hit triggers surrender without killing them; repeated hits do not restart surrender or create extra medical kits. The existing out-of-sight removal remains. Verified on UZDoom 5.0.1: lethal-sized player/environmental hits, repeated explosions and poison, exactly one kit, survival while visible for 12 seconds, and disappearance after ten seconds behind a solid wall. Closing as fixed.

**Comment by hawkwind3** [comment]
I still think, that since surrendered medics cannot be killed, this should apply to ALL surrendered - mechanics, scientists and the surrendered soldier in C3M5_A. They are now disarmed and should not be killable. Killing a surrendered/disarmed enemy soldier is also a war crime. My 2c worth ...
